### ISSUE TITLE
Field Color

### ISSUE THREAD

**Issue by Simashax** [haxball/haxball-issues#160]
What do you think about adding a possibility to change field color? Just like we have a code to change teams' colors, it would be fun to have a code to change the background. It could look like this: /field 000000

**Comment by basro** [comment]
Having it as a custom stadium property would make more sense.

**Comment by Wenomka** [comment]
probably not color but possibility change type of grass, make it another

**Comment by haxballer** [comment]
I think the change in the color of the turf could be not good for our eyes, but changing the shape of the "grass" on the pitch would be fun. :D

**Comment by Leo-MapCreator** [comment]
@basro please make it possible to modify in custom stadiums, and more than only one background, something like layers and you decide the position they will be

**Comment by helppe** [comment]
@simashax good idea, when i created maps i was bored to always use hockey or classic colors.

@haxballer "not good for our eyes" ----> LOL

**Comment by haxballer** [comment]
@helppe You don't understand what I meant. I used to play with colors on my computer and checked various configurations under haxball. At first it looked nice, but during the game the picture was spilled, especially strong, bright colors aren't good for the game. I understand that people like you playing yellow maps are saddened that they can't set a different color, I'm worried mainly about those who prefer traditional haxball, not dumb maps for laymen. (laugh)
